docs/student.md

```diff
@@ -86,6 +86,46 @@ The organizing committee does not recommend that students start research and dev
 
   - According to the final project evaluation results, the remaining project bonus will be paid in proportion.
 
+## Student bonus instruction
+
+1. How much bonus amount will each student receive?
+
+Each project will be awarded RMB 12000, RMB 9000 and RMB 6000 (before tax), which based on the level of project difficulty.
+
+2. When will the bonus be paid?
+
+Students who pass the mid-term assessment will receive 50% of the project bonus first. According to the final project evaluation results, the remaining project bonus will be paid in proportion. If the student fails the final evaluation, the rest bonus will not be paid.
+
+3. How to pay the students’ bonus?
+
+The students’ bonus will be paid by credit transfer method. There is no restriction on which bank but the bank account must be under your own name. Also, the organizing committee will collect the following information from the mentor:
+
+- Name
+
+- ID number
```

Comment:
I wonder what is ID number ? Is this Passport No. ?

Reply:
Sorry , it's a vague expression. is should be 'Passport No.'.